pkg/common/health/config.go

```diff
@@ -0,0 +1,39 @@
+package health
+
+import "time"
+
+type Config struct {
+	Address  string        `hcl:"listen_address"`
+	Interval time.Duration `hcl:"interval"`
```

Comment:
Any thoughts on what circumstances this might be tweaked? I wouldn't mind a hardcoded value here, a user setting this to 1s or less seems like it could be bad

side-thought: any idea what hcl does for time.Duration type? This old issue seems to suggest that duration is expressed as nanoseconds (!)
https://github.com/hashicorp/hcl/issues/202

Reply:
Hardcoding a value seems OK to me too, though I think if we do that, the individual checks should hardcode them (Eg, if somebody pays per request to an upstream CA, you may want to health check every hour instead of every 5 seconds).

Reply:
Hmm yes that's a good point. I don't think that should be too problematic. A plugin implementation with extra constraints like that could also decouple internally

Reply:
Each check now specifies its own interval (though no checks are added yet, except a base dummy one)